src/color/tier.json

```diff
@@ -0,0 +1,71 @@
+{
+    "color": {
+        "tier": {
+            "alaska": {
+                "mvp": {
+                    "onLight": {
+                        "value": "{color.brand.goldgray.200.value}",
+                        "public": true,
+                        "onLight": true,
+                        "usage": "MVP tier color for Alaska Airlines",
+                        "wcag": "n/a",
+                        "deprecated": false
+                    },
+                    "onDark": {
+                        "value": "{color.brand.goldgray.100.value}",
+                        "public": true,
+                        "onDark": true,
+                        "usage": "MVP tier color for Alaska Airlines",
+                        "wcag": "n/a",
+                        "deprecated": false
+                    }
+                },
+                "mvpgold": {
+                    "onLight": {
+                        "value": "{color.brand.gold.200.value}",
+                        "public": true,
+                        "onLight": true,
+                        "usage": "MVP Gold tier color for Alaska Airlines",
+                        "wcag": "n/a",
+                        "deprecated": false
+                    },
+                    "onDark": {
+                        "value": "{color.brand.gold.100.value}",
+                        "public": true,
+                        "onDark": true,
+                        "usage": "MVP Gold tier color for Alaska Airlines",
+                        "wcag": "n/a",
+                        "deprecated": false
+                    }
+                }
+            },
+            "oneworld": {
+                "emerald": {
+                    "value": "{color.brand.emerald.value}",
+                    "public": true,
+                    "onLight": true,
+                    "usage": "Emerald tier color for Oneworld",
+                    "wcag": "n/a",
+                    "deprecated": false
+                },
+                "sapphire": {
+                    "value": "{color.brand.sapphire.value}",
+                    "public": true,
+                    "onDark": true,
+                    "usage": "Sapphire tier color for Oneworld",
+                    "wcag": "n/a",
+                    "deprecated": false
+                },
+                "ruby": {
+                    "value": "{color.brand.ruby.value}",
+                    "public": true,
+                    "onDark": true,
+                    "usage": "Ruby tier color for Oneworld",
+                    "wcag": "n/a",
+                    "deprecated": false
+                }
+            }
+        }
+    }
+}
+  
```

Comment:
This isn't getting registered as the new line at end of file because it has white space that should be removed.
```suggestion

```